lib/widgets/select_semester.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+import 'package:admu_student_app/widgets/select_color.dart';
+
+class SelectSemester extends StatefulWidget {
```

Comment:
i assigned the select semester buttons to marco tho, i added that note in the notion page

Reply:
Yeah, I just made it because its code is similar to the SelectColor class, it can be deleted naman.